package.json

```diff
@@ -45,7 +45,10 @@
     "gulp-concat": "^2.6.0",
     "gulp-connect": "^2.0.6",
     "gulp-eslint": "^3.0.1",
+    "gulp-footer": "^1.0.5",
     "gulp-header": "^1.7.1",
+    "gulp-if": "^2.0.2",
+    "gulp-jscs": "^3.0.2",
```

Comment:
yeah, it was removed [here](https://github.com/prebid/Prebid.js/pull/1111/files#diff-b9e12334e9eafd8341a6107dd98510c9L19) in #1111